Source/WebCore/platform/mediastream/mac/AVVideoCaptureSource.mm

```diff
@@ -296,6 +349,11 @@ static bool isZoomSupported(const Vector<VideoPreset>& presets)
         settings.setZoom(zoom());
     }
 
+    if (supportedWhiteBalanceModes(device()).size()) {
```

Comment:
Would it read better with !isEmpty() instead?

Reply:
Fixed